torch/distributions/__init__.py

```diff
@@ -34,14 +34,15 @@
 from .beta import Beta
 from .categorical import Categorical
 from .cauchy import Cauchy
+from .chi2 import Chi2
```

Comment:
Don't do this. Chi2 imports Gamma, this will create circular import

Reply:
I have changed it. Sorry about that.